src/providers/reversing-distances/__tests__/reversing-distances.spec.ts

```diff
@@ -54,6 +67,11 @@ describe('ReversingDistancesProvider', () => {
         expect(result.startDistance).toEqual(52.5);
       });
 
+      it('should return a value 3 and a half times vehicle length is greater then 16.5', () => {
```

Comment:
This description doesn't quite read right...
Maybe 'should return a value 3 and a half times vehicle length **if** greater **than** 16.5' ?